src/components/vm/overview/cpuModal.tsx

```diff
@@ -116,19 +131,19 @@ export const CPUModal = ({ vm, maxVcpu, models }) => {
         setMax(stateDelta.max);
     }
 
-    function onCountSelect (value) {
-        // Allow empty string
+    function onCountSelect (value: string) {
+        // Allow empty string, which we represent as zero.
         if (value == "") {
-            setCount("");
+            setCount(0);
```

Comment:
This added line is not executed by any test.  [Details](https://logs-cockpit.apps.ocp.cloud.ci.centos.org/pull-2185-af23a7d6-20250716-112918-fedora-42-devel/Coverage/lcov/github-pr.diff.gcov.html)